app/scripts/stores/layout.es6

```diff
@@ -0,0 +1,30 @@
+import { createStore } from 'reflux';
+import SaneStore from '../utils/sane-store-mixin';
+import { toggleCharts } from '../actions/layout';
+import { charts } from '../constants/layout';
+
+
+const LayoutStore = createStore({
+  initialData: {
+    charts: charts.Closed(),
+  },
+  mixins: [SaneStore],
+  init() {
+    this.listenTo(toggleCharts, 'toggleCharts');
+  },
+  update(what, to) {
+    this.setData({
+      ...this.get(),
```

Comment:
It's sort of different, and sort of the same. Here is what the JSX compiles down to with the object spread operator:
```js
const otherProps = {a: 1, b: 2};
<Component prop="val" prop2="val2" {...otherProps} />
```
compiles to
```js
React.createElement(Component, {
  prop: 'val',
  prop2: 'val2',
  ...otherProps
}, []);
```

so it's.... kind of... the same.

The nice thing about it is that it's a short way to create a new object with some things changed from the old one without mutating it. I'm going to pull in immutablejs though, which will enforce that and give us a more normal javascript syntax (and performance improvements and awesome stuff).